src/ripple.js

```diff
@@ -1,47 +1,45 @@
-;(function($, document, Math){
-    $.ripple = function(selector, options) {
+;(function($, Math){
 
-        var self = this;
+    var defaults = {
+        debug: false,
+        on: 'mousedown',
 
-        var _log = self.log = function() {
-            if(self.defaults.debug && console && console.log) {
-                console.log.apply(console, arguments);
-            }
-        };
+        opacity: 0.4,
+        color: "auto",
+        multi: false,
 
-        self.selector = selector;
-        self.defaults = {
-            debug: false,
-            on: 'mousedown',
+        duration: 0.7,
+        rate: function(pxPerSecond) {
+            return pxPerSecond;
+        },
 
-            opacity: 0.4,
-            color: "auto",
-            multi: false,
+        easing: 'linear'
+    };
 
-            duration: 0.7,
-            rate: function(pxPerSecond) {
-                return pxPerSecond;
-            },
+    $(function ($) {
+        $.ripple('[data-ripple-init]');
+    });
+
+    $.ripple = function(selector, options) {
 
-            easing: 'linear'
-        };
+        options = $.extend({}, defaults, options);
 
-        self.defaults = $.extend({}, self.defaults, options);
+        $('body').on(options.on, selector, Trigger);
 
-        var Trigger = function(e) {
+
+        function Trigger(e) {
```

Comment:
It's called hoisting, but in this instance, var Trigger was defined before it was used so there is no difference in switching to function Trigger